app/models/setting/content.rb

```diff
@@ -1,13 +1,13 @@
 class Setting::Content < Setting
```

Comment:
Keeling this class might be a bad idea as it is still creating settings in a DB on app start, so you'd have the settings with old and new category. Do we need to keep it?